en/advanced-topics/jsb-web-difference.md

```diff
@@ -0,0 +1,88 @@
+# The differences between Pure JS and JS-Binding objects
```

Comment:
It is better to describe the main purpose of the documentation, for example

```
Because xxx, if you want your program run on all platforms(especially on native platforms), should not use some yyy features.
```

Reply:
It seems it has already been done at `Possible Solution` section.

Reply:
No. This section just mention what's the purpose of this documentation, no how to do. Just the same as `who need to read this documentation`

Reply:
Yes, that's better. I updated the doc.